### PR TITLE
Memcached effect

### PR DESCRIPTION
**Pull request by possibilities**

### CODE REVIEW

**Comment by ThrivingKings** [comment]
Doesn't it also need to be added to https://github.com/Versal/micro-extensions/blob/master/src/effects/index.js ?

**Comment by possibilities** [comment]
Good call, done

File: src/effects/memcached.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Memcached from 'memcached-promisify'
+
+const memcached = ({
+  memcachedHost,
+  memcachedHosts,
+  memcachedOptions
+}) => {
+  const host = memcachedHost || memcachedHosts
```
Comment (ThrivingKings) [inline src/effects/memcached.js:8]: I'm a little confused by this. How come the singular host is more important? Mainly wondering why the distinction

Comment (possibilities) [reply src/effects/memcached.js:8]: I just want it to be generally useful. It's kinda an interpretation of the underlying API at https://github.com/3rd-Eden/memcached I guess.

**Review by ThrivingKings — APPROVED** [review]

File: src/effects/memcached.js (hunk 1 above)
Comment (possibilities) [reply src/effects/memcached.js:8]: For them it's an un-named option as an argument to an fn. For us it gets named in a config file so I want it to be useful to us in our single-host universe but also be able to take advantage of the underlying lib.

**Pull request merged**